tests/integration/coordinator/plugins/clickhouse/conftest.py

```diff
@@ -38,7 +38,7 @@
 
 logger = logging.getLogger(__name__)
 
-pytestmark = [pytest.mark.clickhouse, pytest.mark.x86_64]
```

Comment:
Why removing the mark ?

Reply:
That was an oversight, was executing integration tests on arm, reverted that chunk. Happy to report that integration tests pass on both architectures.